lib/helper.js

```diff
@@ -145,29 +145,28 @@ function jsxTranspile(instrument) {
     var fs = require('fs');
     var babel = require('babel');
 
-    if (instrument !== undefined && instrument && instrument === 'true') {
-        var istanbul = require('istanbul');
-        var coverageVariable = Object.keys(global).filter(
-            function (key) {
-                return /^\$\$cov_/.test(key);
-            })[0];
-        var instrumenter = new istanbul.Instrumenter({
-                coverageVariable: coverageVariable
-            });
-        require.extensions['.jsx'] = function (module, filename) {
+    require.extensions['.jsx'] = function (module, filename) {
+        var content = fs.readFileSync(filename, 'utf8');
+        var compiled = babel.transform(content, {nonStandard: true, retainLines: true});
+
+        if (instrument && instrument === 'true' && !/(tests|node_modules)/.test(filename)) {
```

Comment:
`instrument && instrument === 'true'` is the same as just `instrument === 'true'` so this this would fail for `instrument = true`. If you need to support strings please do `if (instrument && instrument !== 'false')`